### DISCUSSION TITLE
How can I contribute? I can't push ;/

### DISCUSSION THREAD

**Discussion by omateusanjos** [andreaswilli/react-verification-input#64] [Q&A] [answered]
I made a feature and I can't open a PR ;/

```
mateusanjos@MacBook-Air-de-Mateus react-verification-input % git push --set-upstream origin feat/custom-character-validated
remote: Permission to andreaswilli/react-verification-input.git denied to omateusanjos.
fatal: unable to access 'https://github.com/andreaswilli/react-verification-input.git/': The requested URL returned error: 403
```

Can you help me with this?

**Comment by andreaswilli** [comment, marked as answer]
Thanks for contributing!

You can't push to my repo directly, but you can [open a PR from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).

**Reply by omateusanjos** [reply]
Thank you very much too! Your package saved me these days.
